invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/message.overrides

```diff
@@ -28,3 +28,7 @@
 .ui.message .ui.checkbox {
   margin-top: @defaultMargin / 2.5;
 }
+
+.ui.message.community-branding-error {
+  max-width: 25em;
```

Comment:
what issue does this custom CSS solve?

Reply:
The error renders per each community and this shortens it's size in case error message is huge. Without the max-width it will look like this (with the horizontal scroll to see the Remove button):
<img width="1061" alt="Screenshot 2023-10-02 at 11 33 16" src="https://github.com/inveniosoftware/invenio-app-rdm/assets/61321254/8b21bd46-e534-493c-b338-5f1ae6ba9cad">